samples/dnn/resnet_ssd_face_python.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import argparse
+import os
+import sys
+sys.path.append('/home/arrybn/build/opencv/lib')
+import cv2 as cv
+from cv2 import dnn
```

Comment:
Please avoid local absolute paths.

Consider to add this line into your `.bash_profile` (or `.bashrc`) files:
```
export PYTHONPATH=/home/arrybn/build/opencv/lib
```

and restart your terminal (open new one) or call `source ~/.bash_profile`

Reply:
removed this from the script